assets/blueprints/metadata/src/lib.rs

```diff
@@ -2,16 +2,29 @@ use scrypto::prelude::*;
 
 #[blueprint]
 mod metadata {
+    define_static_auth! {
```

Comment:
Naming is the most difficult. How about `define_auth_template!`, which better captures what it does?

Reply:
I'll defer the naming of this to another PR as I think there are more static things we need to capture in this header macro.